efear:

  robot_id: 1  # 1: car, 0: hand draft 
  # Topics
  pointCloudTopic: "/oculii_radar/point_cloud"
  input_type: "PointCloud2"
  imuTopic: "/imu/data"
  odomTopic: "/odom"                   
  gpsTopic: "/gps/fix1" 
  #gpsTopic: "/ublox/fix"                   # GPS odometry topic from navsat, see module_navsat.launch file

  # Frames
  baselinkFrame: "base_link"
  odometryFrame: "base_link"
  mapFrame: "map"
  
  # Export settings
  # savePCD: false                              # 
  # savePCDDirectory: "/Downloads/4DRadarSLAM/"        #

  # Sensor Settings
  # z_low_thresh: -5.0                         # default: -5.0, minimum z value of the point cloud
  # z_high_thresh: 10.0                          # default: 10.0, maximum z value of the point cloud
  # distance_near_thresh: 1.0                          # default: 1.0, minimum lidar range to be used
  # distance_far_thresh: 100.0                       # default: 100.0, maximum lidar range to be used
  
  # Matching Settings
  registration_type: "P2P"
  resolution: 1.0         # The voxel size for the voxel grid features, in meters 
  enable_dynamic_object_removal: true


  # Extrinsics (Radar -> IMU)
  extrinsicTrans: [0, 0, 0]
  extrinsicRot: [0, 0, 1,
                  -1, 0, 0,
                  0, -1, 0]

ego_velocity_estimator:
  inlier_thresh: 1.5 # m/s, depend on the speed

